gulpfile.js

```diff
@@ -14,7 +14,7 @@ const npmRoot = path.join(repoRoot, 'node_modules')
 const govukCountryPickerDist = path.join(npmRoot, 'govuk-country-and-territory-autocomplete', 'dist')
 const govukToolkitRoot = path.join(npmRoot, 'govuk_frontend_toolkit')
 const govukElementsRoot = path.join(npmRoot, 'govuk-elements-sass')
-const govukFrontendRoot = path.join(npmRoot, 'govuk-frontend')
+const govukFrontendRoot = path.join(npmRoot, 'digitalmarketplace-govuk-frontend/govuk-frontend')
```

Comment:
We should use the power of path.join and not rely on '/' directory seperator
```suggestion
const govukFrontendRoot = path.join(npmRoot, 'digitalmarketplace-govuk-frontend', 'govuk-frontend')
```